### PR TITLE
fix form/input element ID

### LINKED ISSUE #119: Elements with ID equivalent to element name is a code smell
I see this in the serial interaction bit of the UI:

```
<form id="form" action="">
        <input id="input" autocomplete="off"><button>Send</button>
</form>
```

This is a code smell, and a potential footgun if the UI goes through a refactor. Should use more descriptive names that suggest the purpose of the element, not the type of the element.

### PR DESCRIPTION
**Pull request by wtrgjadmw**
fix #119

### CODE REVIEW

**Review by mhiramat — APPROVED** [review]
LGTM

File: docs/servo_power_monitor/src/ui.ts [hunk 1]
```diff
@@ -16,8 +16,12 @@ const analyzeButton = document.getElementById(
...
   'selectDutSerialButton'
 ) as HTMLButtonElement;
-const form = document.getElementById('form') as HTMLFormElement;
-const input = document.getElementById('input') as HTMLInputElement;
+const DutCommandForm = document.getElementById(
+  'DutCommandForm'
+) as HTMLFormElement;
+const DutCommandInput = document.getElementById(
+  'DutCommandInput'
+) as HTMLInputElement;
```
Comment (hikalium) [inline docs/servo_power_monitor/src/ui.ts:24]: they should be lowerCamelCase, not UpperCamelCase
https://google.github.io/styleguide/tsguide.html#naming-rules-by-identifier-type

**Review by hikalium — CHANGES_REQUESTED** [review]

**Comment by wtrgjadmw** [comment]
fixed to lowerCamelCase!

**Review by hikalium — APPROVED** [review]
Looks good! Thanks!!

**Pull request merged**